docs/testing/automation-setup.md

```
@@ -1,123 +1,224 @@
+- Each module is resolved via `MODULE_MAP`
+- Unknown modules are skipped safely
 
-def run_login(page, env):
-        url = "https://dev.example.com" if env == "dev" else "https://pay.example.com"
-        page.goto(url + "/login")
+### Supported Module Keys
 
-        fill(page, "role", ("textbox", "Email"), "test@example.com")
-        fill(page, "role", ("textbox", "Password"), "mypassword")
-        click(page, "role", ("button", "Login"))
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)